packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -1691,117 +1797,136 @@
+						<Button onClick={addSlide}>Add Slide</Button>
+					</div>
+					<div className="flex flex-wrap gap-2">
 						<Input
-							value={item.image}
-							onChange={(e) => updateSlide(index, "image", e.target.value)}
-							placeholder={`Image URL ${index + 1}`}
+							type="number"
+							value={slideCount}
+							onChange={(e) =>
+								onUpdate({
+									slideCount: Math.min(parseInt(e.target.value), items.length),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)